CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ release.
 
 ## Unreleased
 
+## v1.22.0 (2023-10-11)
```

Comment:
Did you verify that no PR slipped through without adding a changelog entry?

Reply:
I didn't look in detail, I'll look at this later today when I'm out of my meeting blocks.